src/app/events/components/search/search.component.ts

```diff
@@ -7,11 +11,39 @@ import { Component, Input, OnInit } from '@angular/core';
 export class SearchComponent implements OnInit {
   @Input() titleColor: string = 'orange';
   @Input() titleText: string = 'Филтриране на събития';
+  @Output() sendData = new EventEmitter<any>();
```

Comment:
Send what data? Variable name is too generic

Reply:
fixet